bundle/Resources/config/routing/admin/field.yml

```diff
@@ -4,6 +4,11 @@ netgen_tags_admin_field_autocomplete:
         _controller: eztags.admin.controller.field:autoCompleteAction
         _method: GET
 
+netgen_tags_add_tag_button_visibililty:
```

Comment:
Can you prefix the route name to `netgen_tags_admin_` to be consistent with other routes? Route path `/access/add` might be more concise, without having to include `netgen` and `tags` in the route name.